lms/djangoapps/course_api/blocks/transformers/blocks_api.py

```diff
@@ -72,4 +73,5 @@ def transform(self, usage_info, block_structure):
         BlockDepthTransformer(self.depth).transform(usage_info, block_structure)
         BlockNavigationTransformer(self.nav_depth).transform(usage_info, block_structure)
         VideoBlockURLTransformer().transform(usage_info, block_structure)
+        VideoBlockStreamPriorityTransformer().transform(usage_info, block_structure)
```

Comment:
Please add dependency, if any, in doc string of this class.

Reply:
Done